.pre-commit-config.yaml

```diff
@@ -61,4 +61,5 @@ repos:
         additional_dependencies:
           - types-pkg_resources
           - rich
-          - types-click
```

Comment:
Typeshed had incorrect typing in one case, and `click` types seem to have moved to the repo itself: https://github.com/pallets/click/pull/1856